pkg/router/template/configmanager/haproxy/manager.go

```diff
@@ -915,8 +928,23 @@ func (entry *routeBackendEntry) BuildMapAssociations(route *routeapi.Route) {
 	}
 }
 
+// validateBlueprint runs extended validation on a blueprint route.
+func validateBlueprintRoute(route *routeapi.Route) error {
+	errs := validation.ExtendedValidateRoute(route)
```

Comment:
Check out [this package](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/errors/errors.go#L29) for easier aggregated error handling.

Reply:
Thanks - will use.